client/client_test.go

```diff
@@ -350,3 +394,76 @@ func (s *clientTestSuite) TestStmt_Trans(c *C) {
 	str, _ = r.GetString(0, 0)
 	c.Assert(str, Equals, `abc`)
 }
+
+// the certificates are in go-mysql/docker/resources
+
+var caPem = []byte(`-----BEGIN CERTIFICATE-----
```

Comment:
I find that many places use the same keys, can we use move them to one place to reduce the code?

Reply:
sure

Reply:
maybe move to the siddontang/mysql folder? but I think it's a bit confusing since the keys are for testing purpose only and the mysql folder is an actual working code base and if we do move the keys there, we have to expose them outside the package and that for users will be an unnecessary leak.

Reply:
I prefer to create a test_util mod which only the tests can use it.

Reply:
ok  that will do

Reply:
already fixed